packages/internal/generated-clients/src/imx-openapi.json

```diff
@@ -4,14 +4,16 @@
   ],
   "swagger": "2.0",
   "info": {
+    "description": "Immutable X API",
     "title": "Immutable X API",
```

Comment:
Looks like the same title and description resulted in the duplicate `Immutable X API` comments in the generated files. Might be something worth fixing in the source when we get a chance.